Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/bean/ConceptsMergeBean.java

```diff
@@ -0,0 +1,164 @@
+package edu.asu.conceptpower.app.bean;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import edu.asu.conceptpower.app.core.POS;
+import edu.asu.conceptpower.core.ConceptType;
+
+public class ConceptsMergeBean {
+
+    public List<String> conceptIds;
+    private Map<String, String> posMap = new LinkedHashMap<String, String>();
+    private Set<String> word;
+    private String selectedPosValue;
+    private String selectedListName;
+    private Set<String> conceptList;
+    private Set<String> description;
+    private Set<String> synonymsids;
+    private Set<String> conceptTypeIdList;
+    private Set<String> equals;
+    private Set<String> similar;
+    private String selectedTypeId;
+    private String selectedTypeName;
+    private ConceptType[] types;
+    private String selectedConceptId;
```

Comment:
all these variables should not be called "word" or "conceptList" if their type is ```Set```. This is very confusing.